tests/test_6_writers.py

```diff
@@ -72,7 +72,7 @@ def test_write_basic():
         assert_array_equal(ImmutMorphology(Path(tmp_folder, "test_write.swc")).points, expected)
         h5_morph = ImmutMorphology(Path(tmp_folder, "test_write.h5"))
         assert_array_equal(h5_morph.points, expected)
-        assert h5_morph.version == ('h5', 1, 2)
+        assert h5_morph.version == ('h5', 1, 3)
```

Comment:
don't worry about changing these tests; once you're happy w/ the API, I'll do the final pass to on tests and such.